File: Juan/P4/Ruby/Napakalaki/lib/CombatResult.rb
#encoding: utf-8

#Versión 3.0
module NapakalakiGame
module CombatResult
    
    WINGAME = :WINGAME
    WIN = :WIN
    LOSE = :LOSE
    LOSEANDCONVERT = :LOSEANDCONVERT
 
    def to_s
      if [CombatResult::WINGAME]          
          return "Has ganado el juego"
      end
                
          if [CombatResult::WIN]
             return "Has ganado"
          end
          
      if [CombatResult::LOSE]
          return "Has perdido"
      end
     
        if [CombatResult::LOSEANDCONVERT]
            return "Te conviertes"
        end
      
    end
end
end

File: Juan/P5/java/Napakalaki/src/NapakalakiGame/CombatResult.java
package NapakalakiGame;

/**
 * @version 3.1
 * @author juane
 */
public enum CombatResult {
    WINGAME,WIN,LOSE, LOSEANDCONVERT;
    
    
    @Override
    public String toString(){
        
        if(this.equals(WIN)){
            
            return "Ganas";
        }
        else if (this.equals(LOSE)){
            
            return "Pierdes";
        }
        
        else if(this.equals(WINGAME)){
           
            return "WINGAME";
        }
        else if(this.equals(LOSEANDCONVERT)){
           
            return "CultistPlayer";
        }
        else
            return "HA OCURRIDO UN ERROR";
    }

   

  
    
}
